Basic/2DSpaceShooter/README.md

```diff
@@ -1,14 +1,31 @@
-## 2D Space Shooter
+# 2D Space Shooter
 
-This is a UNet sample project converted to Netcode for GameObjects.
+![UnityVersion](https://img.shields.io/badge/Unity%20Version:-2021.3%20LTS-57b9d3.svg?logo=unity&color=2196F3)
+![NetcodeVersion](https://img.shields.io/badge/Netcode%20Version:-1.2.0-57b9d3.svg?logo=unity&color=2196F3)
+[![LatestRelease](https://img.shields.io/badge/Latest%20%20Github%20Release:-v1.2.0-57b9d3.svg?logo=github&color=brightgreen)](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/releases/tag/v1.2.0)
+<br><br>
 
-## Sample Features
+This is a UNet sample project converted to Netcode for GameObjects. The 2DSpaceShooter sample is a bitesize sample designed to demonstrate networked 2D and physics-based character movement. 
+<br><br>
 
-This sample shows you:
+# Sample Overview
 
-- Server authorative physics based movement
-- Managing health and a list of buffs for your players in a multiplayer game
+In this sample, learn more about:
+
+- Server authorative physics based movement using Netcode for GameObject's `NetworkRigidbody2D` component
+- Managing health and a list of buffs for your players in a multiplayer game with a `NetworkVariable`
 - How to pool network objects such as bullets and asteroids to improve performance
+<br><br>
+
+## Exploring the Sample
+
+The entry scene for this game is the network scene. From there a game can be hosted or an existing game can be joined.
```

Comment:
Should we add a couple more sentences here about what you can do in the scene and where things are?

Reply:
good call!

Reply:
these have been added in :)